core/domain/collection_domain.py

```diff
@@ -303,10 +303,11 @@ def to_dict(self):
 
     @classmethod
     def create_default_collection(
-            cls, collection_id, title=feconf.DEFAULT_COLLECTION_TITLE,
-            category=feconf.DEFAULT_COLLECTION_CATEGORY,
-            objective=feconf.DEFAULT_COLLECTION_OBJECTIVE,
-            language_code=constants.DEFAULT_LANGUAGE_CODE):
+            cls, collection_id: str,
+            title: str =feconf.DEFAULT_COLLECTION_TITLE,
+            category: str =feconf.DEFAULT_COLLECTION_CATEGORY,
+            objective: str =feconf.DEFAULT_COLLECTION_OBJECTIVE,
+            language_code: str =constants.DEFAULT_LANGUAGE_CODE):
```

Comment:
Why do you type this?

Reply:
because in `caching_services_test.py` we are calling this function in typed manner https://github.com/oppia/oppia/blob/6431446ad481100a8d71d79ab5636c0e6c72aef6/core/domain/caching_services_test.py#L750

Reply:
If it is just in test then I think you can add ignore to the test file which will be removed later.

Reply:
done!